language/bytecode_verifier/bytecode_verifier_tests/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
```

Comment:
Could you turn these tests into unit tests?

Reply:
for sure!